lib/publiccloud/instance.pm

```diff
@@ -432,9 +502,11 @@ sub softreboot {
     my $start_time = time();
 
     # wait till ssh disappear
-    while (($duration = time() - $start_time) < $args{timeout}) {
-        last unless (defined($self->wait_for_ssh(timeout => 1, proceed_on_failure => 1, username => $args{username})));
-    }
+    my $out = $self->wait_for_ssh(timeout => $args{timeout}, wait_stop => 1, username => $args{username});
+    # ok ssh port closed
+    record_info("STOP NOK", "WARNING: while stopping the system, ssh port still open after timeout,\nreporting: $out")
```

Comment:
```suggestion
    record_info("Shutdown failed", "WARNING: while stopping the system, ssh port still open after timeout,\nreporting: $out")
```

I assume NOK means not-ok. Let's please not use slang, which might be confusing.